netmaster/master/network.go

```diff
@@ -163,6 +164,57 @@ func deleteDockNet(networkName string) error {
 	return nil
 }
 
+func detachServiceContainer(networkName, tenantName string) error {
```

Comment:
Sorry if this is orthogonal to this PR and may be this discussion need to happen somewhere else, so kindly correct me. But I think by wrapping up service-discovery container inside netmaster we are tying state to a specific netmaster instance. For example, how will service-discovery work if this particular netmaster instance were to get killed and another  netmaster instance now needs to take over?